README.md

```diff
@@ -170,6 +172,35 @@ You can also use `ca: self-sign` or `ca: local` depending on your certmonger usa
 
 Note that this does *not* work on RHEL/CentOS 7.
 
+#### Generate a new certificate in the role
+
+You can also use the `certificate` role inside the `cockpit` role to create
+certificates by providing `cockpit_certificates`.
```

Comment:
This is redundant with the paragraph below, and "providing" is confusing. If we drop the current approach from the section above, then this paragraph can just be dropped.